### PR TITLE
r/aws_securityhub: Add aws_securityhub_member resource

### PR DESCRIPTION
**Pull request by gazoakley**
Partly addresses #6674

Changes proposed in this pull request:

* **New Resource:** `aws_securityhub_member`

Output from acceptance testing:

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSSecurityHub'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAWSSecurityHub -timeout 120m
=== RUN   TestAccAWSSecurityHub
=== RUN   TestAccAWSSecurityHub/Account
=== RUN   TestAccAWSSecurityHub/Account/basic
=== RUN   TestAccAWSSecurityHub/Member
=== RUN   TestAccAWSSecurityHub/Member/invite
=== RUN   TestAccAWSSecurityHub/Member/basic
=== RUN   TestAccAWSSecurityHub/ProductSubscription
=== RUN   TestAccAWSSecurityHub/ProductSubscription/basic
=== RUN   TestAccAWSSecurityHub/StandardsSubscription
=== RUN   TestAccAWSSecurityHub/StandardsSubscription/basic
--- PASS: TestAccAWSSecurityHub (185.81s)
    --- PASS: TestAccAWSSecurityHub/Account (27.28s)
        --- PASS: TestAccAWSSecurityHub/Account/basic (27.28s)
    --- PASS: TestAccAWSSecurityHub/Member (70.77s)
        --- PASS: TestAccAWSSecurityHub/Member/invite (46.90s)
        --- PASS: TestAccAWSSecurityHub/Member/basic (23.87s)
    --- PASS: TestAccAWSSecurityHub/ProductSubscription (51.00s)
        --- PASS: TestAccAWSSecurityHub/ProductSubscription/basic (51.00s)
    --- PASS: TestAccAWSSecurityHub/StandardsSubscription (36.75s)
        --- PASS: TestAccAWSSecurityHub/StandardsSubscription/basic (36.75s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	185.852s
```

### CODE REVIEW

**Comment by gazoakley** [comment]
Actually, I'll probably tweak this to include better error information once https://github.com/aws/aws-sdk-go/issues/2372 is resolved

**Comment by scrofungulus** [comment]
This has been open for a while. What will it take to get this merged in?

**Comment by scrofungulus** [comment]
@gazoakley ^^

File: website/docs/r/securityhub_member.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# aws_securityhub_member
+
+Provides a Security Hub member resource.
+
+## Example Usage
+
+```hcl
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_member" "example" {
```
Comment (gdavison) [inline website/docs/r/securityhub_member.markdown]: Should this include a `depends_on` argument?

**Comment by gazoakley** [comment]
@gdavison Rebased and should be up to date now

**Comment by anuj-modi** [comment]
What is the status on this PR? Really looking forward to this feature along with a corresponding acceptor.

**Comment by cteig** [comment]
I'm also waiting for this one. What does it take to get this merged?

**Comment by anuj-modi** [comment]
@gdavison What work still needs to be done for this PR to be accepted?

**Comment by mhmdio** [comment]
@gdavison look like all require work is done, Can this be merged? LGTM.

**Comment by frodeb** [comment]
Patiently waiting for this feature to be merged as well :-)

**Pull request merged**